database/migrations/2021_12_09_075535_create_products_details_table.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Support\Facades\DB;
+
+class CreateProductsDetailsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        DB::statement("
+        CREATE TABLE products_details (
```

Comment:
I recommend a bit to work on indentation.

Reply:
Corrected. I will pay attention on it in the future.